src/project/git/GitCommandGitProject.ts

```diff
@@ -69,11 +78,16 @@ export class GitCommandGitProject extends NodeFsLocalProject implements GitProje
                          directoryManager: DirectoryManager = DefaultDirectoryManager): Promise<GitCommandGitProject> {
         return clone(credentials, id, opts, directoryManager)
             .then(p => {
-                const pathIntoRepo = !!id.path ?
-                    id.path.startsWith("/") ? id.path : "/" + id.path :
-                    "";
-                const gp = GitCommandGitProject.fromBaseDir(id, p.baseDir + pathIntoRepo, credentials);
-                return gp;
+                if (!!id.path) {
```

Comment:
this is Rod's stuff for supporting projects in subdirs inside Repos. I want to change this a bit but not before this PR is merged.